config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_logginglogmetrics.logging.cnrm.cloud.google.com.yaml

```diff
@@ -71,7 +76,6 @@ spec:
                       bounds:
                         description: The values must be monotonically increasing.
                         items:
-                          format: double
```

Comment:
IIRC, we were told this info is not exactly sent over to the api server but 🙈 we still want these, right?

Reply:
Yes, we believe this is not actually used.  That we specify double (float64) but the official tools drop it supports this.  So I don't think we need it.

Maybe we should try to figure out how to add it just so we don't need to worry about it, but it would be a pain I think...

Reply:
Actually, scratch that, just found the `// +kubebuilder:validation:Format` directive

Reply:
Although this particular one is a problem because the format has moved to the slice, which is likely wrong

Reply:
I like using kubebuilder to validate the CRD. Also, we shall have` // +kubebuilder:validation:Required ` on `spec.filter`.

One thing I don't figure out how it works is that the [LLM CRD has `filter` set as required](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/crds/logging_v1beta1_logginglogmetric.yaml#L389). But I was assuming the CRD is generated from this file which should have `filter`  as `optional` (default). Thoughts?